rust/foxglove/src/websocket_server.rs

```diff
@@ -175,6 +174,18 @@ impl WebSocketServerHandle {
             .await;
     }
 
+    /// Publishes a status message to all clients.
+    /// https://github.com/foxglove/ws-protocol/blob/main/docs/spec.md#status
```

Comment:
Needs a bit more formatting for rustdoc. How about:

```suggestion
    ///
    /// For more information, refer to the [Status][status] message specification.
    ///
    /// [status]: https://github.com/foxglove/ws-protocol/blob/main/docs/spec.md#status
```